### ISSUE TITLE
[CLI] Parser error with regexp containing $ character

### ISSUE THREAD

**Issue by shalak** [pomber/git-history#90]
Reproduction:
1. Commit the following file:
```python
def _parse_uppercase(self):
    p = re.compile(r'^([A-Z0-9_]+)+_.*$')
    m = p.match(self._name())
    return m.group(1) if m else self._name()
```
2. Open via `git-file-history`
3. The history page is broken: no commiters and only some parser-leftover code. Then the Git History README is being shown:
![image](https://user-images.githubusercontent.com/1536554/52768963-3d352500-302f-11e9-8123-53dfa78a46de.png)

**Comment by pomber** [comment]
Thanks, I think I fixed this in #88. I'll let you know when I publish a new version.

**Comment by pomber** [comment]
Could you try with the new version? `npx git-file-history@0.0.4 your/file`

**Comment by shalak** [comment]
It didn't help. With `0.0.4`, only the README is shown.

**Comment by shalak** [comment]
@pomber - Does #98 fix this one? How can I run the most recent version?

**Comment by pomber** [comment]
@shalak I just published it `npx git-file-history@0.1.0 ./foo.bar`
